charts/smartmetserver/Chart.yaml

```diff
@@ -11,14 +10,12 @@ description: Helm chart to deploy SmartMet Server for Kubernetes
 # a dependency of application charts to inject those utilities and functions into the rendering
 # pipeline. Library charts do not define any templates and therefore cannot be deployed.
 type: application
-
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 version: 0.1.0
```

Comment:
Chart Version needs to be updated

Reply:
Updated. See: https://github.com/fmidev/helm-charts/commit/122b121e1082afd61184275f8a4c8c2c46e99f56